zapcore/field_test.go

```diff
@@ -164,6 +165,10 @@ func TestFields(t *testing.T) {
 }
 
 func TestEquals(t *testing.T) {
+	timeOutOfRange := time.Unix(0, math.MaxInt64).Add(time.Nanosecond)
+	timeOutOfRangeNano := time.Unix(0, timeOutOfRange.UnixNano())
```

Comment:
We should try out of range in both directions.

Reply:
:+1: done